wordpress-mobile/WordPress-iOS
# Gutenberg/715 image upload cancelation

Updates the app to support image cancelation when a block is deleted on the Gutenberg editor.

To test:
 - Open or create a post with GB editor active
 - Add an image block
 - Select an image from the device.
 - While the image is uploading delete the image block
 - Check on the media library that the image was not added to the media library.

Update release notes:
